qpdf fails to cross-compile

### Steps To Reproduce
Steps to reproduce the behavior:
1. `nix build .#pkgsCross.aarch64-multiplatform.qpdf`

### Build log
```
checking for "/dev/urandom"... configure: error: cannot check for file existence when cross compiling
```

### Metadata

```console
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.132, NixOS, 22.05 (Quokka), 22.05.20220728.9370544`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.8.1`
 - channels(yvesf): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/development/libraries/qpdf/default.nix
+++ b/pkgs/development/libraries/qpdf/default.nix
@@ -15,6 +15,9 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ zlib libjpeg ];
 
+  configureFlags = lib.optional (stdenv.hostPlatform != stdenv.buildPlatform)
+                     "--with-random=/dev/urandom";
+
   preCheck = ''
     patchShebangs qtest/bin/qtest-driver
   '';

